public/mockServiceWorker.js

```diff
@@ -0,0 +1,303 @@
+/* eslint-disable */
```

Comment:
I assume all of this is a paste

Reply:
This file is generated by CLI (https://mswjs.io/docs/getting-started/integrate/browser#setup)